core/modules/widgets/dropzone.js

```diff
@@ -27,14 +27,14 @@ DropZoneWidget.prototype = new Widget();
 Render this widget into the DOM
 */
 DropZoneWidget.prototype.render = function(parent,nextSibling) {
-	var self = this;
```

Comment:
No, not at all, it's just that there are plenty of those extraneous declarations. If we care about them, we should probably be applying lint and doing them in bulk. It's just to try to keep the pull requests simple to understand, and it is a general observation: in many specific cases where a function is being heavily modified these kind of modifications don't get in the way. In this particular case it just makes the fairly simple mods look more complex at first sight.